### PR TITLE
#50: Add typing for Scale create

### PR DESCRIPTION
**Pull request by kopecmi8**

### CODE REVIEW

**Comment by kopecmi8** [comment]
https://github.com/JsCommunity/human-format/issues/50

File: index.d.ts [hunk 1]
```diff
@@ -65,9 +67,17 @@ export declare namespace humanFormat {
     ): ParsedInfo<S>;
   }
 
+  class Scale {
+    constructor(prefixes: Record<string, number>)
+  }
+
+  namespace Scale {
+    function create<S extends ScaleName>(prefixesList: Array<string>, base: number, initExp?: number): Scale
```
[the remark this answers is not in the document]
Comment (kopecmi8) [reply index.d.ts:75]: you are right, it was there because I had previously S as return type, but its not neede, after reworked solution

**Comment by ByScripts** [comment]
Hi,

Thank you very much for your participation.

I improved it in this PR: #52 

It will allow to suggest and validate `prefix` property based on `Scale` configuration.

**Comment by julien-f** [comment]
@kopecmi8 Thank you for your contribution :slightly_smiling_face: 

@ByScripts, the author of the initial type declarations has opened an alternative PR #52, feel free to review and comment there :slightly_smiling_face:

**Comment by julien-f** [comment]
Superseded by #52

**Pull request closed without merging**